src/style.css

```diff
@@ -1 +1,75 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+   
+}
+
+body {
+  margin: 0;
+}
+
+.logo_block {
+  width: 40px;
+  height: 40px;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li:nth-child(-n+7) {
+  margin-right: 20px;
+}
```

Comment:
avoid global styling by tag 

What happened if I wanna use li in other palaces?